doc/articles/features/focus-management.md

```diff
@@ -15,3 +15,17 @@ WinRTFeatureConfiguration.Focus.EnableExperimentalKeyboardFocus = true;
 ```
 
 The feature requires additional testing to verify all edge cases on Android and iOS. In a future release, we will switch the experimental support to be enabled by default.
+
+## Disabling initial focus on Page
+
+The focus management logic in UWP/WinUI sets initial focus on `Page` that is being loaded (for example during navigation) if no element in the app is currently focused. This may cause an input element like `TextBox` to get focused automatically. This may not be desirable behavior though, as it will cause the virtual keyboard on mobile platforms to open up. To avoid this initial focus, please set `IsTabStop` of the `Page` to `false`:
+
+```xaml
+<Page
+   ...
+   IsTabStop="False">
+   ...
+</Page>
+```
+
+Alternatively you can set focus explicitly to some other UI element by calling `element.Focus(FocusState.Programmatic)` in the code-behind.
```

Comment:
You might want to specify at exactly what point this should be called? (Eg from the ctor? On Loaded? Some other moment?)